Gemfile

```diff
@@ -4,5 +4,8 @@ group :test do
   gem 'rake'
   gem 'rspec'
   gem 'beaker', '~>1.17.0'
+  if ENV['GEM_SOURCE'] =~ /rubygems\.delivery\.puppetlabs\.net/
```

Comment:
@cprice404 
What do you think of this? I orignally just tested `GEM_SOURCE` being set since that is how other projects I've observed do it. I'm hesitant to add an otherwise meaningless variable but I think this addresses both concerns.